st2common/st2common/util/pack.py

```diff
@@ -43,9 +43,15 @@
 ]
 
 # Common format for python 2.7 warning
-PACK_PYTHON2_WARNING = "DEPRECATION WARNING: Pack %s only supports Python 2.x. " \
+if six.PY2:
+    PACK_PYTHON2_WARNING = "DEPRECATION WARNING: Pack %s only supports Python 2.x. " \
                        "Python 2 support will be dropped in future releases. " \
                        "Please consider updating your packs to work with Python 3.x"
+else:
+    PACK_PYTHON2_WARNING = "DEPRECATION WARNING: Pack %s only supports Python 2.x. " \
+                       "Python 2 support has been removed. " \
```

Comment:
Do you think it worth mentioning when the python 2 removal happened in the platform? Might give some users hints if they want to downgrade the st2 version.

```suggestion
                       "Python 2 support has been removed since st2 v3.4.0. " \
```